Model/Game.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace GameOfLife
+{
+    [Serializable]
+    public class Game
+    {
+        /// <summary>
+        /// Properties.
+        /// </summary>
+        public int Height { get; set; }
+        public int Width { get; set; }
+        public int[,] currentGeneration { get; private set; }
+        public int iterationCount = 1;
+
+        /// <summary>
+        /// Constructor without parameters.
+        /// </summary>
+        public Game()
+        {
+            Height = 16;
```

Comment:
It is bad to have number literals in code. They should be put in some kind of repository.
You can create static class with name something like ApplicationParameters, define constants there and in code use those constants instead of number literals.

Reply:
Static class GameParameters with 2 constant variables created.